Resources/ArmTemplates/template.json

```diff
@@ -116,23 +122,6 @@
     ]
   },
   "resources": [
-    {
```

Comment:
it's probably worth removing the other storage account once this update is deployed

Reply:
will do